examples/events_sse_transport/main.go

```diff
@@ -18,11 +18,9 @@ package main
 
 import (
 	"flag"
-	"time"
-
 	marathon "github.com/gambol99/go-marathon"
-
-	"github.com/golang/glog"
+	"log"
+	"time"
```

Comment:
Oh wow. That's cool. I only used gofmt so far. Added goimports to the stack now.